.github/workflows/main.yml

```diff
@@ -53,6 +58,16 @@ jobs:
               flavor: rockylinux
               major: 9
             java_major: 17
+          - tomcat_major: 10
+            base_image:
+              flavor: rockylinux
+              major: 8
+            java_major: 21
+          - tomcat_major: 10
+            base_image:
+              flavor: rockylinux
+              major: 9
+            java_major: 21
```

Comment:
I think we only want this one, the previous os and tomcat are still build for retrocompat of previous releases